pom.xml

```diff
@@ -49,7 +49,7 @@
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
-            <version>4.10</version>
+            <version>4.11</version>
```

Comment:
This fixes a compilation error (#30) when initially building the library (at least it did for me).